packages/bpk-component-dialog/src/BpkDialog.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import BpkModal from 'bpk-component-modal';
+
+const BpkDialog = props => {
+  const {
+    isOpen,
+    onClose,
+    target,
+    closeOnScrimClick,
+    showHeader,
+    ...rest
+  } = props;
+
+  return (
+    <BpkModal
+      isOpen={isOpen}
+      onClose={onClose}
+      target={target}
+      closeOnScrimClick={closeOnScrimClick}
+      showHeader={showHeader}
+      fullScreenOnMobile={false}
+      {...rest}
+    />
+  );
+};
+
+BpkDialog.propTypes = {
+  isOpen: PropTypes.bool.isRequired,
+  onClose: PropTypes.func.isRequired,
+  target: PropTypes.element,
+  closeOnScrimClick: PropTypes.bool,
```

Comment:
Dude I think we wanna give this a better API - make the configurations first class. I suggest we hide the implementation detail of `closeOnScrimClick` and `showHeader` behind a better name. There are 2 configs right? Default and "not dismissable". The latter of which effectively require the consumer to provide a control within the modal to bind the `onClose` to...

Reply:
Done. Changed `closeOnScrimClick` & `showHeader` to `dismissible`. Also in the stories 'not dismissible' example it is specified that the consumer needs to provide control within the dialog to close it.